.dependabot/config.yml

```diff
@@ -0,0 +1,14 @@
+version: 1
+update_configs:
+  # Keep package.json (& lockfiles) up to date as soon as
+  # new versions are published to the npm registry
+  - package_manager: "javascript"
+    directory: "/"
+    update_schedule: "live"
+
+    # Apply default reviewer and label to created
+    # pull requests
+    default_reviewers:
+      - "vamcart"
```

Comment:
Could you please set all core developers here.

Reply:
Okay... But after the merge, due to conflict errors.

Reply:
By the way you sure that it's not working for you with yarn, woks very good for me.

Reply:
Yarn was for me fine, the problem yesterday was npm build.